rc/filetype/kotlin.kak

```diff
@@ -0,0 +1,277 @@
+    add-highlighter -override shared/kotlin/code/java_exceptions_B regex \b($(join "${java_exceptions_B}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_C regex \b($(join "${java_exceptions_C}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_D regex \b($(join "${java_exceptions_D}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_E regex \b($(join "${java_exceptions_E}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_F regex \b($(join "${java_exceptions_F}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_G regex \b($(join "${java_exceptions_G}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_H regex \b($(join "${java_exceptions_H}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_I regex \b($(join "${java_exceptions_I}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_J regex \b($(join "${java_exceptions_J}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_K regex \b($(join "${java_exceptions_K}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_L regex \b($(join "${java_exceptions_L}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_M regex \b($(join "${java_exceptions_M}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_N regex \b($(join "${java_exceptions_N}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_O regex \b($(join "${java_exceptions_O}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_P regex \b($(join "${java_exceptions_P}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_R regex \b($(join "${java_exceptions_R}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_S regex \b($(join "${java_exceptions_S}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_T regex \b($(join "${java_exceptions_T}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_U regex \b($(join "${java_exceptions_U}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_V regex \b($(join "${java_exceptions_V}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_W regex \b($(join "${java_exceptions_W}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_X regex \b($(join "${java_exceptions_X}" '|'))\b 0:rgb:d26937
+    add-highlighter -override shared/kotlin/code/java_exceptions_Z regex \b($(join "${java_exceptions_Z}" '|'))\b 0:rgb:d26937
```

Comment:
We tend to avoid hardcoded rgb faces in filetype scripts, a named face, ideally one of the shared names defined in `colors/default.kak` would be better (here I'd expect the `type` face to be appropriate)